examples/fodo/analysis_fodo_envelope.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022-2023 ImpactX contributors
+# Authors: Axel Huebl, Chad Mitchell
+# License: BSD-3-Clause-LBNL
+#
+
+import argparse
+import glob
+import re
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+def read_file(file_pattern):
+    for filename in glob.glob(file_pattern):
+        df = pd.read_csv(filename, delimiter=r"\s+")
```

Comment:
Double check: reading NaN columns seems to cause issues in other columns.
Example: `sigma_t` column in pandas has issues, but should be well defined.